extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1238,11 +1244,11 @@ export class AmpA4A extends AMP.BaseElement {
   /**
    * Gets the Ad URL to send an XHR Request to.  To be implemented
    * by network.
-   * @param {?CONSENT_POLICY_STATE} unusedConsentState
+   * @param {!ConsentTupleDef} unusedConsentTuple
```

Comment:
I know all network need to override the `getAdUrl()` method, and for most of them `consentTuple` is unused. 
What do you think about making the `consentTuple` optional when `getConsentPolicy` returns null. Instead of calling `getAdUrl({})`, you call `getAdUrl()`. In this case you don't need to change all the tests either.

Reply:
I made this required following [this comment](https://github.com/ampproject/amphtml/pull/27780#discussion_r409927148), thoughts?

Reply:
Looks like the value is never nullable, but can be undefined. 
What about
```
@param {!ConsentTupleDef=} opt_ununsedConsentTuple
```